data/sql_updates/large_aggregates.sql

```diff
@@ -0,0 +1,228 @@
+drop table if exists large_aggregates_tmp;
+create table large_aggregates_tmp as
+-- candidates
+with cand as (
+    select
+        extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+        extract(year from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as year,
+        ttl_receipts_per as ttl_receipts,
+        pol_pty_cmte_contb_per as pty_cmte_contb,
+        other_pol_cmte_contb_per as oth_cmte_contb,
+        ttl_disb_per as ttl_disb,
+        -- double check var
+        tranf_to_other_auth_cmte_per as tranf_to_other_auth_cmte,
+        offsets_to_op_exp_per as offsets_to_op_exp,
+        ttl_loans_received_per as ttl_loan_repymts,
+        ttl_contb_ref_per as ttl_contb_ref,
+        other_disb_per
+    from fec_vsum_f3p
```

Comment:
@LindsayYoung looking over the sql that Paul put together and it used disclosure.v_sum_and_det_sum_report to build this part (I think it's the same part).  He's seen this postgres sql so I'm assuming it's fine, but it is a difference.  Just kind of proofreading stuff.